lib/policyEvaluator/RequestContext.js

```diff
@@ -277,7 +283,7 @@ class RequestContext {
             obj.apiMethod, obj.awsService, obj.locationConstraint,
             obj.requesterInfo, obj.signatureVersion,
             obj.authType, obj.signatureAge, obj.securityToken, obj.policyArn,
-            obj.action);
+            obj.action, obj.quotaDelta);
```

Comment:
Not following how is this being used?

Reply:
This is bad. I fixed it. Changed it to needQuota.
Great catch, thank you!